static/arguments.yml

```diff
@@ -219,6 +219,10 @@
 - component: ButtonGroup
   source: https://github.com/primer/view_components/tree/main/app/components/primer/button_group_component.rb
   parameters:
+  - name: variant
+    type: Symbol
+    default: "`Primer::ButtonComponent::DEFAULT_VARIANT`"
```

Comment:
I _think_ we evaluate constants to their values in other docs, perhaps we could do the same here? That way this would read `:medium`